config/rbac/role.yaml

```diff
@@ -413,3 +413,13 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - storage.k8s.io
+  resources:
+  - storageclasses
+  verbs:
+  - get
+  - list
+  - patch
```

Comment:
I don't think we need to patch `storageclasses` objects 😄

Reply:
fixed